packages/account/src/Sections/Security/LoginHistory/login-history-list-row.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classNames from 'classnames';
+import { TLoginHistoryItems } from 'Types';
+import { Table } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import getLoginHistoryTableHeaders from 'Constants/get-login-history-table-headers';
+import ListCell from './list-cell';
+import { observer, useStore } from '@deriv/stores';
+
+const LoginHistoryListRow = observer(({ id, date, action, browser, ip, status }: TLoginHistoryItems) => {
+    const { date_title, browser_title, action_title, ip_title, status_title } = getLoginHistoryTableHeaders();
+    const { ui } = useStore();
+    const { is_desktop } = ui;
+    return (
+        <div className={classNames('login-history__list__wrapper')} key={id}>
+            <Table.Row className='login-history__list__row login-history__list__row--with-margin'>
+                <Table.Cell className='login-history__list__row__cell'>
+                    <ListCell title={date_title} text={date} />
+                </Table.Cell>
+            </Table.Row>
+            <Table.Row className='login-history__list__row login-history__list__row--with-margin'>
+                <Table.Cell className='login-history__list__row__cell'>
+                    <ListCell
+                        className='login-history__list__row__cell--browser'
+                        title={browser_title}
+                        text={
+                            browser === 'Unknown' ? (
+                                <Localize i18n_default_text='{{browser}}' values={{ browser }} />
```

Comment:
Can we hardcode the browser value to to be `Unknown` here rather than passing it as a value?